app/views/activities/_bounce_activity.html.erb

```diff
@@ -0,0 +1,4 @@
+<li class="activity-feed__item t-activity" id="activity-<%= bounce_activity.id %>">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+  <b>Delivery failure</b> <em><%= bounce_activity.message %></em> <em class="badge"><%= time_ago_in_words(bounce_activity.created_at) %> ago</em>
```

Comment:
Do we get the mailgun headers here? It might be useful to state [which message](https://github.com/guidance-guarantee-programme/planner/blob/master/app/lib/mailgun_headers.rb#L11) it was that bounced?